### ISSUE TITLE
scheduled backup failed on gcp using tidb-cluster helm chart

### ISSUE THREAD

**Issue by noprom** [pingcap/tidb-operator#1563]
## Question
I am using the chart `tidb-cluster` provided by `tidb-operator`, and install it one gcp using helm.
When I set the option `scheduledBackup.create` to `true`, the backup job failed with the logs below:
```
kubectl -n $NS logs -f tidb-scheduled-backup-manual-dkdjl-tdqnv
```
The log:
```
date: invalid date '@dkdjl'
```

And describe the pod:
```
kubectl -n $NS describe pod tidb-scheduled-backup-manual-dkdjl-tdqnv
```
```
  Type     Reason                  Age                    From                                                     Message
  ----     ------                  ----                   ----                                                     -------
  Normal   Scheduled               6m15s                  default-scheduler                                        Successfully assigned tidb/tidb-scheduled-backup-manual-dkdjl-tdqnv to gke-app-cluster-default-pool-cccc96b1-vr3f
  Normal   SuccessfulAttachVolume  6m9s                   attachdetach-controller                                  AttachVolume.Attach succeeded for volume "pvc-17aa4742-5a8e-4b62-a125-26f88515ab34"
  Normal   Pulling                 6m5s                   kubelet, gke-app-cluster-default-pool-cccc96b1-vr3f  Pulling image "pingcap/tidb-cloud-backup:20191217"
  Normal   Pulled                  5m50s                  kubelet, gke-app-cluster-default-pool-cccc96b1-vr3f  Successfully pulled image "pingcap/tidb-cloud-backup:20191217"
  Normal   Created                 4m17s (x5 over 5m48s)  kubelet, gke-app-cluster-default-pool-cccc96b1-vr3f  Created container scheduled-backup
  Normal   Started                 4m17s (x5 over 5m48s)  kubelet, gke-app-cluster-default-pool-cccc96b1-vr3f  Started container scheduled-backup
  Normal   Pulled                  4m17s (x4 over 5m47s)  kubelet, gke-app-cluster-default-pool-cccc96b1-vr3f  Container image "pingcap/tidb-cloud-backup:20191217" already present on machine
  Warning  BackOff                 58s (x24 over 5m46s)   kubelet, gke-app-cluster-default-pool-cccc96b1-vr3f  Back-off restarting failed container
```

Any ideas about why it failed?

**Comment by aylei** [comment]
Which tidb-cluster chart version are you using?

**Comment by noprom** [comment]
@aylei The chart version is `tidb-cluster-v1.0.1`.

**Comment by aylei** [comment]
That version requires special pod name format to work, the pod name you posted is weird that it has a word `manual` in it and it does not contain the backup timestamp. That's why the backup did not work.

It is recommended to upgrade tidb-operator and the charts to the latest `v1.0.x` version.

**Comment by noprom** [comment]
@aylei Appreciated, problem solved by upgrading to `v1.0.6`.
